src/NuGet.Clients/PackageManagement.UI/Models/SearchResultPackageMetadata.cs

```diff
@@ -4,41 +4,161 @@
 
+        // Indicates whether the instance is created for solution package manager
+        public bool IsSolution { get; set; }
+
+        public string Author { get; set; }
+
         public NuGetVersion Version { get; set; }
 
+        private NuGetVersion _installedVersion;
+
+        // The installed version of the package.
+        public NuGetVersion InstalledVersion
+        {
+            get
+            {
+                return _installedVersion;
+            }
+            set
+            {
+                if (!VersionEquals(_installedVersion, value))
+                {
+                    _installedVersion = value;
+                    OnPropertyChanged(nameof(InstalledVersion));
+                }
+            }
+        }
+
+        // The latest version of the package in the current source if
+        // installed version is not the latest.
+        private NuGetVersion _latestVersion;
+
+        public NuGetVersion LatestVersion
+        {
+            get
+            {
+                return _latestVersion;
+            }
+            set
+            {
+                if (!VersionEquals(_latestVersion, value))
+                {
+                    _latestVersion = value;
+                    OnPropertyChanged(nameof(LatestVersion));
+                }
+            }
+        }
+
+        private bool VersionEquals(NuGetVersion v1, NuGetVersion v2)
+        {
+            if (v1 == null && v2 == null)
+            {
+                return true;
+            }
+
+            if ((v1 == null && v2 != null) ||
+                (v1 != null && v2 == null))
+            {
+                return false;
+            }
+
+            return v1.Equals(v2, VersionComparison.Default);
+        }
+
+        private int? _downloadCount;
+
+        public int? DownloadCount
+        {
+            get
+            {
+                return _downloadCount;
+            }
+            set
+            {
+                _downloadCount = value;
+                if (_downloadCount.HasValue && _downloadCount.Value > 0)
+                {
+                    double v = _downloadCount.Value;
+                    int exp = 0;
+                    while (v > 1000)
+                    {
+                        v /= 1000;
+                        ++exp;
+                    }
+
+                    _downloadCountText = string.Format(
+                        CultureInfo.CurrentCulture,
+                        "{0:G3}{1}",
+                        v,
+                        _scalingFactor[exp]);
+                }
+                else
+                {
+                    _downloadCountText = string.Empty;
+                }
+
+                OnPropertyChanged(nameof(DownloadCountText));
+            }
+        }
+
+        private string _downloadCountText;
+
+        public string DownloadCountText
+        {
+            get
+            {
+                return _downloadCountText;
+            }
+        }
+
```

Comment:
Added properties Author, DownloadCount, InstalledVersion, LatestVersion since they are required by the new UI design.